xarray/backends/rasterio_.py

```diff
@@ -141,7 +141,19 @@ def open_rasterio(filename, chunks=None, cache=None, lock=None):
         # CRS is a dict-like object specific to rasterio
         # We convert it back to a PROJ4 string using rasterio itself
         attrs['crs'] = riods.crs.to_string()
-    # Maybe we'd like to parse other attributes here (for later)
+    if hasattr(riods, 'res'):
+        # (width, height) tuple of pixels in units of CRS
+        attrs['res'] = riods.res
+    if hasattr(riods, 'tiled') or \
+       (hasattr(riods, 'profile') and 'tiled' in riods.profile):
+        # Is the TIF tiled? (bool)
+        # We cast it to an int for netCDF compatibility
+        attrs['tiled'] = np.uint8(riods.tiled if hasattr(riods, 'tiled')
```

Comment:
Ah, good catch. It turns out that the attribute is actually called `is_tiled`: https://mapbox.github.io/rasterio/api/rasterio.io.html#rasterio.io.DatasetReader . Just pushed up the correction.